docs/hip-16.md

```diff
@@ -63,29 +56,44 @@ do not autorenew, and are always removed from the ledger when they expire.
 
 ## Backwards Compatibility
 
-There is no change in existing protobufs. Account and entity owners must ensure that linked autorenew and admin accounts have
-sufficient balances for autorenewal fees, or risk permanent removal of their entity! The Hedera Product team will set and
-publicize the timeline for enabling the autorenewal and autoremoval behaviors.
+There is no change in existing protobufs. Account and entity owners must ensure that linked autorenew and admin accounts have sufficient balances for autorenewal fees, or risk permanent removal of their entity.
+
+Every account will receive one free auto renewal at implementation of this feature. This will have the effect of extending the initial period for autorenewal ~92 days.
 
 ## Security Implications
 
-N/A
+A Hedera Account without an hbar balance sufficient to cover the cover the cost of renewal at the point of renewal would be marked for deletion. The Account owner then have a grace period to fund the account and renewal before permanent deletion.
```

Comment:
A Hedera Account with zero balance at the point of renewal would be marked for deletion.

A Hedera Account with non-zero balance that is not sufficient to cover the entire cost of renewal will have its remaining balance wholly used for a shorter extension of the entity.